README.md

```diff
@@ -1,3 +1,66 @@
+<!-- markdownlint-disable-next-line -->
+![Logo](.github/ASSETS/logo.png)
+
+[![Version](https://img.shields.io/spiget/version/117279?label=version)][spigotmc-link]
+[![Discord](https://img.shields.io/discord/185055040036143104?color=404eed&label=Discord&logo=discord&logoColor=ffffff)][discord-invite]
+[![View on SpigotMC](https://img.shields.io/badge/view%20on-SpigotMC-orange.svg)][spigotmc-link]
+[![Downloads](https://img.shields.io/spiget/downloads/117279?color=blue)][spigotmc-link]
+
 # UltraCosmetics GUI Addon
 
-Helper plugin for the [UC-GUI resource pack](https://github.com/dennishzg/uc-gui). Addon for the main plugin, [UltraCosmetics](https://github.com/datatags/UltraCosmetics).
+UltraCosmetics GUI Addon is a [UltraCosmetics](https://github.com/datatags/UltraCosmetics) addon for [UC-GUI resource pack](https://github.com/dennishzg/uc-gui)!
+
+---
+
+## ? Compatibility
```

Comment:
Are these supposed to be emojis like in the main UC readme? They're not rendering in the preview for me

Reply:
Thanks Windows... Fixed!